app/assets/javascripts/modules/cookies.js

```diff
@@ -0,0 +1,32 @@
+moj.Modules.cookies = {
+  read(name) {
+    const nameEQ = `${encodeURIComponent(name)}=`;
+    const cookies = document.cookie.split(/\s*;\s*/);
+    let cookieValue;
+
+    cookies.forEach((cookie) => {
+      if (cookie.indexOf(nameEQ) === 0) {
+        cookieValue = decodeURIComponent(cookie.substring(nameEQ.length, cookie.length));
```

Comment:
One small thing - this continues looping through the cookies after it has found the one you want. You could use `cookies.find(cookie => cookie.startsWith(nameEQ))` instead.

Reply:
Ooh, like it. 👍